phase2/ignition/vanilla/manifest/kube-apiserver.jsonnet

```diff
@@ -35,6 +35,7 @@ function(cfg)
               "--tls-cert-file=/srv/kubernetes/apiserver.pem",
               "--tls-private-key-file=/srv/kubernetes/apiserver-key.pem",
               "--secure-port=443",
+              "--storage-backend=etcd2", 
```

Comment:
I propose to use etcd3 by default. Cause etcd2 has limitations in aspects. e.g. memory, storage. ETCD3 has big changes between etcd2 and etcd3. There're flags in kubernetes to set the image  to be used, and there're flags to set the storage version to be choosen.
https://github.com/kubernetes/kubernetes/blob/master/cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest#L57
https://github.com/kubernetes/kubernetes/blob/master/cluster/saltbase/salt/etcd/etcd.manifest#L41

What needed to be done is to set the kube-up parameters to set up pillars for the salt state to support them.